stash_engine/app/models/stash_engine/resource.rb

```diff
@@ -470,7 +470,7 @@ def fill_author_from_user!
     # Publication
     # Files are published when the publication date has been reached
     def files_published?
-      metadata_published? && publication_date.present? && Date.today.to_s >= publication_date
+      metadata_published? && publication_date.present? && Time.new >= publication_date
```

Comment:
Good enhancement. We may want to do this in other places as well (for example we capture publication_date as a Date '2019-04-25' it may be a good idea to ensure we convert it to a time)

Reply:
I removed these.